docs/api.rst

```diff
@@ -111,7 +112,7 @@ Accepts an object representing options with the values to updated. Other options
 highlightAll
 ------------
 
-Applies highlighting to all ``<pre><code>...</code></pre>`` blocks on a page.
+Applies highlighting to all elements on a page matching ``allCodeSelector``.
```

Comment:
```suggestion
Applies highlighting to all elements on a page matching the configured ``cssSelector``.
The default ``cssSelector`` value is ``pre code``, which highlights all code blocks. 
```

Reply:
> highlights all code blocks

This phrasing seems to suggest that `<code>` tags _not_ wrapped in `<pre>` are **not code blocks**; would that be a good justification to remove `display: block;` from the `.hljs` rule (possibly migrating it to a rule for `pre .hljs`) in a future PR?

Reply:
> to suggest that <code> tags not wrapped in <pre> are not code blocks;

They aren't - they are just inline code.  A few words of code does not a block make.  We could add:

```
If you wish to highlight inline `<code>` elements also, try `cssSelector: "code"`.
```

Or some such... to make it clearer.

Reply:
I think that makes it clear that block is used in the CSS sense of the the meaning.

Reply:
> They aren't - they are just inline code.

I take it, then, that you would be amenable to (in a future PR) removing or moving [the line that transforms them into blocks](https://github.com/highlightjs/highlight.js/blob/11.0.0-beta0/src/styles/default.css#L11-L12)?